{NA <type 'exceptions.TypeError'> "'str' object is not callable"
 Traceback (most recent call last):
  File "regression-tests/uncallable.py", line 2, in <module>
    x = s()
}
